src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,78 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Interpretation for edges of GEOMETRY logical type, i.e. whether the edge
+ * between points represent a straight cartesian line or the shortest line on
+ * the sphere.
+ */
+enum Edges {
+  PLANAR = 0;
+  SPHERICAL = 1;
+}
+
+/**
+ * A custom WKB-encoded geometry data to be used in geometry statistics.
+ * The geometry may be a polygon to encode an s2 or h3 covering to provide
+ * vendor-agnostic coverings, or an evelope of geometries when a bounding
+ * box cannot be built (e.g. a geometry has spherical edges, or if an edge
+ * of geographic coordinates crosses the antimeridian).
+ */
+struct Geometry {
+  /** Bytes of a WKB-encoded geometry */
+  1: required binary geometry;
+  /**
+   * Edges of the geometry if it is a polygon. It may be different to the
+   * edges attribute from the GEOMETRY logical type.
+   */
+  2: optional Edges edges;
```

Comment:
might be worth specifying whether this is logically required or if a default is assumed.

Reply:
I'm inclined to make it `required` because the geometry is anyway a polygon which has this attribute. Though writer who generates this geometry statistics is required to fill the `Edges` field, it makes the reader easier in that it does not need to derive this attribute from logical type. Sometimes the values in the geometry column do not have any polygon values, which means the logical type does not need to write the `Edges` field. cc @paleolimbot

Reply:
Is it better to use `bool is_spherical` instead of an enum `Edges edges` given the fact that it has only two options? @paleolimbot